### PR TITLE
Add new commands to man file

### PR DESCRIPTION
**Pull request by ddotldot**

### CODE REVIEW

File: mesaflash.c [hunk 1]
```diff
@@ -193,9 +194,9 @@ void print_usage() {
     printf("  --rpo             Read hostmot2 variable directly at 'address'.\n");
     printf("  --wpo             Write hostmot2 variable directly at 'address'\n");
     printf("                    with 'value'.\n");
-    printf("  --set             Set board IP address in EEPROM to n.n.n.n (only\n");
+    printf("  --set             Set board IP address in EEPROM memory to n.n.n.n (only\n");
```
Comment (jepler) [inline mesaflash.c:197]: Should this (and one other reference above) be changed to FLASH memory like the other references, or is there a separate EEPROM for the board IP address?

**Review by jepler — COMMENTED** [review]
Thank you for keeping the documentation up to date with your changes. I did have one question.

File: mesaflash.c (hunk 1 above)
Comment (ddotldot) [reply mesaflash.c:197]: No, it really EEPROM memory chip(93C46) used for storage IP, MAC and board name.
FPGA configuration stored in FLASH memory chip.

Comment (jepler) [reply mesaflash.c:197]: thanks, that is helpful clarification for me.

**Pull request merged**